spec/features/bookmarks_spec.rb

```diff
@@ -58,26 +58,26 @@
   end
 
   it "cites items in current bookmarks page" do
-    visit solr_document_path('2009373513')
+    visit solr_document_path('2009373513') # Ci an zhou bian
     click_button 'Bookmark'
 
-    visit solr_document_path('2007020969')
+    visit solr_document_path('2007020969') # Strong Medicine speaks
     click_button 'Bookmark'
 
     visit "/bookmarks?per_page=1"
-    expect(page).to have_content 'Ci an zhou bian'
-    expect(page).not_to have_content 'Strong Medicine speaks'
+    expect(page).to have_content 'Strong Medicine speaks'
+    expect(page).not_to have_content 'Ci an zhou bian'
```

Comment:
I'm a bit surprised to see the change that we're making affect this test.  Is there a particular reason that the sort changed here?